apps/game_client/lib/game_client/client_socket_handler.ex

```diff
@@ -48,7 +53,6 @@ defmodule GameClient.ClientSocketHandler do
   end
 
   defp ws_url(player_id, game_id) do
-    # TODO Remove hardcoded host
```

Comment:
Why removing this TODO tag? 
If we want to remove it we should remove it in `apps/game_client/assets/js/hooks/game_queue.js` as well

Reply:
Fixed